velox/exec/tests/SparkAggregationFuzzerTest.cpp

```diff
@@ -36,8 +36,9 @@ DEFINE_string(
     "(e.g: --only \"min\" or --only \"sum,avg\").");
 
 int main(int argc, char** argv) {
+  std::string prefix = "spark_";
```

Comment:
Why do we need the prefix? I assume this change is unrelated. Let's extract it into a separate PR.